agenthub/codeact_agent/action_parser.py

```diff
@@ -158,6 +161,33 @@ def parse(self, action_str: str) -> Action:
         return AgentDelegateAction(agent='BrowsingAgent', inputs={'task': task})
 
 
+class CodeActActionParserFileEdit(ActionParser):
+    """Parser action:
+    - FileEditAction(diff_block) - Search/Replace block to edit.
+    """
+
+    def __init__(
+        self,
+    ):
+        self.diff_block = None
+
+    def check_condition(self, action_str: str) -> bool:
```

Comment:
Where is this used?

Reply:
See parse_action in CodeActResponseParser